half way through the first draft

alpatania opened the pull request.

alpatania committed:
Seconda Traduzione

alpatania committed:
Terza Traduzione

linea 1067 potrebbe essere troppo lunga (7 extra char)
linea 1078 non so se e' ben tradotta (exposure therapy -> terapia espositiva)
linea 1088 la traduzione e' un po' claudicante, e' una delle scelte da fare per il lupo quindi c'e' pochissimo spazio
linea 1102 Non so bene come tradurlo
---
in pausa linea 1118 perche' non so come tradurre "snuff murder porn"

antalele commented on scenes/act4.md:
```diff
@@ -428,86 +428,2006 @@ h: Okay. Healthy relationship with emotions. Relationships need communication. S
+
+`bb({ mouth:"scream", eyes:"scream", body:"two_up" })`
+
+b: Soon we'll give ourselves treats while watching snuff murder porn!
```
presto ci daremo piacere guardando pornografia sodomitica!

alpatania replied:
da una ricerca su internet pero' lo snuff murder porn non e' proprio quello - e' vedere gente che viene uccisa in diretta/in video e torturata per davvero, non a fini di intrattenimento porno. Alla fine il BDSM consensuale e' un fetish sano, ma qui parlano di diventare psicopatici e dei serial killer (piu' in la' nella traduzione). non vorrei mettere il BDSM consensuale in quella categoria.

antalele commented:
tradurrei la linea 1118 come: presto ci daremo piacere guardando pornografia sodomitica!

alpatania committed:
Merge branch 'gh-pages' into gh-pages

alpatania committed:
Update act4.md

linea 770, 793, 824 ha un fill automatico in base alle scelte del gioco. bisogna controllare che la grammatica fili.
Acute Accents: á, é, í, ó, ú
